src/main/java/net/earthcomputer/clientcommands/features/PlayerRandCracker.java

```diff
@@ -424,8 +435,7 @@ public enum Type {
             NOT_ENOUGH_ITEMS(false, "playerManip.notEnoughItems"),
             NOT_POSSIBLE(false, "playerManip.throwError"),
             UNKNOWN_SEED(false, "playerManip.uncracked"),
-            SUCCESS(true, null),
-            ;
+            SUCCESS(true, null);
```

Comment:
The reason it was like how it was before is because it makes the diff cleaner when you add more types

Reply:
I see, do you want me to revert the change?

Reply:
Yes please